angular/README.md

```diff
@@ -1,10 +1,25 @@
-# MyApp
+# WxCC Widget Starter - Angular
```

Comment:
i would keep the work in progress disclaimer that is in the current angular repo

Reply:
I thought you had intentionally removed it, will add it back